packages/common-all/package.json

```diff
@@ -44,6 +45,7 @@
     "luxon": "^1.25.0",
     "minimatch": "5.1.0",
     "nanoid": "^3.1.23",
+    "neverthrow": "^5.0.0",
```

Comment:
moving forward, we should sync before introducing new dependencies or paradigms.
we already have a way of handling errors across our code base with [[dendron://dendron.docs/dev.process.errors#respv3]]

Reply:
[`d0b93c6` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/d0b93c61cf65a9f3add05ac9ff6368ec7ef9d6b2)

I removed `neverthrow` and added some utils around `RespV3`. 

> moving forward, we should sync before introducing new dependencies or paradigms.

yes, I agree. I have been too eager of wanting that paradigm shift. thats something in me that I appreciate but it tends to overshoot. it needs some gently okayness ;) hope I have not risked to much struggle.